Library/Homebrew/cmd/update-report.rb

```diff
@@ -136,6 +140,21 @@ def update_report
     Commands.rebuild_commands_completion_list
     link_completions_manpages_and_docs
     Tap.each(&:link_completions_and_manpages)
+
+    return if new_repository_version.blank?
+
+    ohai "Homebrew was updated to version #{new_repository_version}"
+    puts <<~EOS
+      The changelog can be found at:
+        #{Formatter.url("https://github.com/Homebrew/brew/releases/tag/#{new_repository_version}")}
+    EOS
+
+    return unless new_repository_version.split(".").last == "0"
+
+    puts <<~EOS
+      More detailed release notes are available on the Homebrew Blog:
+        #{Formatter.url("https://brew.sh/blog/")}
```

Comment:
I wonder if it's trying to setup the redirects such that we can link to it directly?

Reply:
Is this existing functionality? If not, any guidance for how to do this?

Reply:
Yeh, GitHub Pages can do this for us. See the existing use of `redirect_from` in https://github.com/Homebrew/brew.sh/blob/master/_posts/2016-09-21-homebrew-1.0.0.md. Could ensure that we always have `blog/1.0.0` or something work as a redirect (and maybe enforce with CI but that's not a blocker/requirement).

Reply:
Is it unreasonable to have `blog/latest-release` always redirect to the latest release notes? This would definitely be enforced by CI but would still require manually changing old blog posts whenever there's a new one. Again, this seems to be mostly you who does the work so if you're fine with it I think it will be okay.

Reply:
I was thinking ideally we'd have `blog/1.0.0` redirect to the post for that and we output the actual `blog/#{tag}` here. I don't mind going back and backfilling all those redirects 😁

Reply:
Ah, gotcha. That's much easier to implement 😁